docs/contributor/concepts/error-output-format.md

```diff
@@ -0,0 +1,42 @@
+# Output Error Format
```

Comment:
When we have many ADR's in the same directory it's easier when we put sequence number or date to file name, i.e.: `001-error-output-format.md` or `20240318-error-output-format.md`.